carsus/io/output/base.py

```diff
@@ -629,6 +632,19 @@ def create_collisions(self, temperatures=np.arange(2000, 50000, 2000)):
 
         return collisions
 
+    def create_cross_sections(self):
+        """
+        """
+        logger.info('Ingesting photoionization cross-sections.')
+
+        cross_sections = self.cmfgen_reader.cross_sections
+        cross_sections['energy'] = u.Quantity(cross_sections['energy'], u.Unit('Ry')).to('eV').to(u.Hz, equivalencies=u.spectral()).value
```

Comment:
Why extra steps between Ry and Hz?
```suggestion
        cross_sections['energy'] = u.Quantity(cross_sections['energy'], u.Unit('Ry')).to(u.Hz, equivalencies=u.spectral()).value
```